tonic/Cargo.toml

```diff
@@ -32,8 +32,8 @@ transport = [
     "tower-balance",
     "tower-load",
 ]
-openssl = ["openssl1", "tokio-openssl", "tls"]
-rustls = ["tokio-rustls", "tls"]
+openssl = ["openssl1", "tokio-openssl", "openssl-probe", "tls"]
+rustls = ["tokio-rustls", "tls", "webpki-roots"]
```

Comment:
I think this might make more sense? So that users could enable it like so: `["rustls", "rustls-roots"]`. I would probably turn this off by default? Not sure.
```toml
rustls-roots = ["webpki-roots"]
openssl-roots = ["openssl-probe"]
```

Reply:
That seems like a reasonable way of dealing with it actually, I can use the same for client identity. Something of a shame Cargo doesn't allow for dependencies iff two features are specified at the same time.

Reply:
Yeah that part is unfortunate but I think if we document it, its an easy cost to pay.